src/gui/editors/AutomationEditor.cpp

```diff
@@ -38,7 +36,10 @@
 #include <QScrollBar>
 #include <QStyleOption>
 #include <QToolTip>
+#include <cmath>
 
+#include "MidiClip.h"
```

Comment:
This one should be at next to `MainWindow.h`.

Reply:
Done